plugin/fuzz/source/dextool/plugin/fuzz/xml_parse.d

```diff
@@ -0,0 +1,457 @@
+module xml_parse;
+
+import std.algorithm;
+import std.array;
+import std.container.array;
+import std.file;
+import std.path;
+import std.string;
+import std.xml;
+import std.typecons;
+
+import std.stdio;
+
```

Comment:
I recommend making the whole module "@safe".
It will _probably_ allow most of the other code to be safe too.

To make the destructor safe in the structs replace all usage of Array with the builtin arrays.
```d
Arra!Primitive prims;
// by
Primitive[] prims;
```

The insertBack is then replace by a ~=.

Why? Then the following bug would have been found by the compiler (see [1]).